src/ACEtk/block/TabulatedOutgoingEnergyDistribution/src/ctor.hpp

```diff
@@ -0,0 +1,42 @@
+TabulatedOutgoingEnergyDistribution() = default;
+
+TabulatedOutgoingEnergyDistribution( const TabulatedOutgoingEnergyDistribution& ) = default;
+TabulatedOutgoingEnergyDistribution( TabulatedOutgoingEnergyDistribution&& ) = default;
+
+/**
+ *  @brief Constructor
+ *
+ *  @param[in] incident        the incident energy value
+ *  @param[in] interpolation   the interpolation type
+ *  @param[in] cosines         the cosine values (N values)
+ *  @param[in] pdf             the pdf values (N values)
+ *  @param[in] cdf             the cdf values (N values)
+ *  @param[in] discrete        the number discrete photon lines (defaults to 0)
+ */
+TabulatedOutgoingEnergyDistribution( double incident,
+                                     int interpolation,
+                                     std::vector< double >&& cosines,
+                                     std::vector< double >&& pdf,
+                                     std::vector< double >&& cdf,
+                                     std::size_t discrete = 0 ) :
+  TabulatedProbabilityDistribution(
+        "DLW::TabulatedOutgoingEnergyDistribution",
+        discrete * 10 + interpolation, std::move( cosines ),
```

Comment:
[clang-tidy - recommended]
discrete is `size_t`, destination is `int`. Suggest making consistent.